scripts/rollup/build.js

```diff
@@ -363,6 +364,22 @@ function getPlugins(
     }),
     // We still need CommonJS for external deps like object-assign.
     commonjs(),
+    isFBBundle && {
```

Comment:
This is not necessary since your enabled externalLiveBindings, right?

Reply:
Yes, I believe so, I will remove it.